packages-ts/gauntlet-terra/src/commands/internal/terra.ts

```diff
@@ -130,6 +134,38 @@ export default abstract class TerraCommand extends WriteCommand<TransactionRespo
     return this.wrapResponse(res)
   }
 
+  async migrateContract(contract: AccAddress, newCodeId: number, migrateMsg: any) {
+    logger.info(`Migrating contract ${contract} to new code id ${newCodeId}`)
+    const msg = new MsgMigrateContract(this.wallet.key.accAddress, contract, newCodeId, migrateMsg)
+    const tx = await this.wallet.createAndSignTx({
+      msgs: [msg],
+      memo: `Migrating ${contract} to ${newCodeId}`,
+      ...(this.wallet.key instanceof LedgerKey && {
+        signMode: SignMode.SIGN_MODE_LEGACY_AMINO_JSON,
+      }),
+    })
+    logger.loading(`Migrating contract...`)
+    const res = await this.provider.tx.broadcast(tx)
+
+    return this.wrapResponse(res)
+  }
+
+  async migrateCode(codeId: number, newCode: string) {
```

Comment:
Same comment as above for `migrateContract` function, but additionally this fn is not used anywhere. Consider implementing it as a command.

Reply:
For now we have no expectation of using this, I'll wait for implementing the command

Reply:
Could you please add a comment that this is an unused piece of code that needs to be moved to its own command?